js/anims-missiles.js

```diff
@@ -0,0 +1,94 @@
+//global scope
+var columnID = 0
+var gameScore = {
+    score:0,
+    toll:0
+}
+
+function dropMissile(animationTime) {
+    var columnNumber = randomNumGen(4)
+    var $missile = $('<img src="img/missile-drop-down-white.png" ' +
+        'class="missile missile-'+ columnNumber +'">')
+    $('#pos-'+columnNumber).prepend($missile)
+
+    $missile.animate({
+        top: "+=550"
+    }, animationTime, "linear", function () {
+        missileHitsCity()
+        $(this).remove()
```

Comment:
why not put this inside missileHitsCity()? you will always want to remove the missile when it hits

Reply:
Fixed in 80513ff

Had to pass the $(this) value into missileHitsCity() as 'that' because 'this' is a reserved word and therefore cannot pass 'this' as a parameter.